GoogleUtilities/Environment/GULAppEnvironmentUtil.m

```diff
@@ -179,6 +179,8 @@ + (NSString *)deviceSimulatorModel {
         model = @"iOS Simulator (Unknown)";
         break;
     }
+#elif TARGET_OS_VISION
```

Comment:
I think moving `#elif TARGET_OS_VISION` above `#elif TARGET_OS_IOS` would allow this to work on both Xcode 15.0 and Xcode 15.1 Beta.

Note: In the current ordering, it'll go in the `TARGET_OS_IOS` path on Xcode 15.0 where the value of `TARGET_OS_IOS` is still `1` on visionOS.